tests/conftest.py

```diff
@@ -29,6 +31,20 @@
 
 from tests.fixtures.explorer.data_contributions import data_contribution
 
+# Betamax config
+
+with betamax.Betamax.configure() as cfg:
+    cfg.cassette_library_dir = 'tests/cassettes/'
+
+    record_mode = 'none' if os.environ.get('CI') else 'once'
+    cfg.default_cassette_options['record_mode'] = record_mode
+    cfg.default_cassette_options['match_requests_on'] = [
+        'uri',
+        'method',
```

Comment:
I just realized this is probably why the tests fail: the match on `method` fails because the [processor makes a `HEAD` request] while betamax made a `GET` request.